FindClangFormat.cmake

```diff
@@ -0,0 +1,30 @@
+#
+#.rst:
+# FindClangFormat
+# ---------------
+#
+# The module defines the following variables
+#
+# ``CLANG_FORMAT_EXECUTABLE``
+#   Path to clang-format executable
+# ``CLANG_FORMAT_FOUND``
+#   True if the clang-format executable was found.
```

Comment:
Since you have added a `FindClangFormat.cmake` file it should also report version information as specified in the [find_package](https://cmake.org/cmake/help/v3.0/command/find_package.html) docs.

* `CLANGFORMAT_VERSION_MAJOR`
* `CLANGFORMAT_VERSION_MINOR`
* `CLANGFORMAT_VERSION_PATCH`
* `CLANGFORMAT_VERSION_COUNT`